client/src/infographic/Infographic.js

```diff
@@ -274,49 +274,12 @@ class InfoGraph_ extends React.Component {
       </div>
     );
   }
-  loadBubbleMenuDeps(subject) {
-    return ensure_loaded({
-      subject: subject,
-      has_results: true,
-    }).then(() => get_panels_for_subject(subject));
-  }
-
-  loadGraphDeps({ panel_keys_by_bubble_id, active_bubble_id, subject, level }) {
-    const panel_keys = panel_keys_by_bubble_id[active_bubble_id];
-
-    ensure_loaded({
-      panel_keys,
-      subject_level: level,
-      subject: subject,
-      footnotes_for: subject,
-    }).then(() => {
-      if (
-        shallowEqualObjectsOverKeys(
-          { active_bubble_id, subject, level },
-          this.state,
-          ["subject", "active_bubble_id", "level"]
-        )
-      ) {
-        const valid_panel_objs = _.chain(panel_keys)
-          .map((key) => {
-            const panel_obj = PanelRegistry.lookup(key, level);
-            return panel_obj.is_panel_valid_for_subject(subject) && panel_obj;
-          })
-          .filter()
-          .value();
-        const valid_subject_panels_by_bubble_id = panel_keys
-          ? {
-              ...panel_keys_by_bubble_id,
-              [active_bubble_id]: _.map(valid_panel_objs, "key"),
-            }
-          : panel_keys_by_bubble_id;
-
+  load({ subject, level, active_bubble_id }) {
```

Comment:
I should have commented on this. Earlier I was saying it should be two different loading steps, but then I realized that half of the bubble menu loading step was not necessary. I think ultimately this giant `Infographic` component should be broken up more at which point there will be two distinct loading steps, but I collapsed them together for now because I was in a bit of a rush.